Makefile

```diff
@@ -12,6 +12,8 @@ ENV_FILE := .env
 
 PROJECT_PATH = $(shell pwd)
 RML_MAPPER_PATH = ${PROJECT_PATH}/.rmlmapper/rmlmapper.jar
+API_HOST = localhost
```

Comment:
This should definetly not be in the Makefile.
Can you add them into Vault? 
Also please follow the ENV variable naming conventions